### ISSUE TITLE
[docs] *Ingester modules are of the producer type not ingester.

### ISSUE THREAD

**Issue by rudo-thomas** [seznam/slo-exporter#65]
This is a terminology/naming bug.

The [architecture documentation](https://github.com/seznam/slo-exporter/blob/63b0a1ed6f439206d03d82867470618d6c75ca84/docs/architecture.md) defines three types of modules: *producer*, *ingester*, *processor*.

The [prometheusIngester](https://github.com/seznam/slo-exporter/blob/63b0a1ed6f439206d03d82867470618d6c75ca84/docs/modules/prometheus_ingester.md) and [kafkaIngester](https://github.com/seznam/slo-exporter/blob/63b0a1ed6f439206d03d82867470618d6c75ca84/docs/modules/kafka_ingester.md) modules include the word *ingester* in their names, despite being *processor* type modules. While this is correctly documented at the modules' page, **it is also very confusing** and should be fixed.

**Comment by FUSAKLA** [comment]
Yes, I agree it's confusing. The naming collision is a bit unfortunate.

Since module names are used in the configuration, changing it would mean a breaking change.
I think changing the pipeline module type names is the way to go.

How about:
 - `producer` -> `source`
 - `processor` (we can leave the same?)
 - `ingester` -> `sink`
 
 @lksv @david-vavra WDYT?
